docs/diagrams/SelectSequenceDiagram.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+Actor User as user USER_COLOR
+Participant ":UI" as ui UI_COLOR
+Participant ":Logic" as logic LOGIC_COLOR
+Participant ":Model" as model MODEL_COLOR
+Participant ":Storage" as storage STORAGE_COLOR
+
+user -[USER_COLOR]> ui : "select 1 2 3 4"
+activate ui UI_COLOR
+
+ui -[UI_COLOR]> logic : execute("select 1 2 3 4")
+activate logic LOGIC_COLOR
+
+logic -[LOGIC_COLOR]> model : selectPersons(indexes)
+activate model MODEL_COLOR
+
+model -[MODEL_COLOR]-> logic
+deactivate model
+
+logic -[LOGIC_COLOR]> storage : saveSelectedIndexes(addressBook)
+activate storage STORAGE_COLOR
```

Comment:
both the selectPerson in model and saveSelectedIndexes in storage needs to be an actual function. Is it done in seperate pr?

Reply:
So we need to keep this updated in 1.5